docs/driver-parameters.md

```diff
@@ -34,6 +34,42 @@ nodeStageSecretRef.namespace | namespace where the secret is | k8s namespace  |
 kubectl create secret generic smbcreds --from-literal username=USERNAME --from-literal password="PASSWORD"
 ```
 
+### Kerberos ticket support for Linux
+#### All the pre-requisites to make it work
+ - Each node must have kerberos support set up, and cifs-utils installed
+ - `/var/lib/kubelet/kerberos/` directory must exist - it'll contain kerberos credential cache files for different users.
+   - This is directory that is shared by smb container and the host.
+ - Each node will be responsible for cleaning that directory as admin see fit. **Unmount does not delete the cache file.**
```

Comment:
can you rephrase these words in chatGpt, this sounds like non native english

Reply:
Done